File: CPP/Code/Schedule.cpp
#include <iostream>
using namespace std;

class Schedule {
private:
    string day;
    string time;
    string room;

public:
    Schedule(string day, string time, string room) : day(day), time(time), room(room) {}

    //fungsi print schedule
    void printSchedule() const {
        cout << "- " << day << " at " << time << " (Room: " << room << ")" << endl;
    }
};

File: Jawa/Code/Schedule.java
public class Schedule {
    private String day;
    private String time;
    private String room;

    public Schedule(String day, String time, String room) {
        this.day = day;
        this.time = time;
        this.room = room;
    }

    //print schedule
    public void printSchedule() {
        System.out.println("- " + day + " at " + time + " (Room: " + room + ")");
    }
}